src/frameworks/CoreServices/include/AE/AEDataModel.h

```diff
@@ -47,4 +47,11 @@ typedef FourCharCode                    AEEventClass;
 typedef FourCharCode                    AEEventID;
 typedef SInt8                           AEArrayType;
 
+typedef OSErr (*AEEventHandlerProcPtr)(
+	const AppleEvent * theAppleEvent,
+	AppleEvent * reply,
+	SRefCon handlerRefcon
+);
```

Comment:
```suggestion
typedef OSErr (*AEEventHandlerProcPtr)(
	const AppleEvent* theAppleEvent,
	AppleEvent* reply,
	SRefCon handlerRefcon
);
```

I'm also okay with this too if you prefer this format:
```suggestion
typedef OSErr (*AEEventHandlerProcPtr)(
	const AppleEvent *theAppleEvent,
	AppleEvent *reply,
	SRefCon handlerRefcon
);
```